web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -99,6 +99,8 @@ export namespace MenuOverlay {
       return this.maxHeight ? `max-height: ${this.maxHeight};` : `max-height: calc(100vh - 48px);`;
     }
 
+    shouldWrapFocus = () => this.ariaRole === 'dialog';
```

Comment:
where are u using this?

Reply:
The `shouldWrapFocus` function is used in the focusTrap method in FocusTrapMixin file, to determine whether focus should wrap around to the first focusable or not based on role is dialog or not